exp/api/v1beta1/azuremanagedmachinepool_types.go

```diff
@@ -189,6 +511,10 @@ type AzureManagedMachinePoolSpec struct {
 	// KubeletConfig specifies the kubelet configurations for nodes.
 	// +optional
 	KubeletConfig *KubeletConfig `json:"kubeletConfig,omitempty"`
+
+	// LinuxOSCustomConfig specifies the custom linux OS settings and configurations
+	// +optional
+	LinuxOSCustomConfig *LinuxOSCustomConfig `json:"linuxOsCustomConfig,omitempty"`
```

Comment:
why not follow the same pattern as kubeletConfig and call it `LinuxOSConfig`
internally in Azure SDK its also `containerservice.LinuxOSConfig`

Reply:
Fair point, will update it!

Reply:
Added via 841f2b4140b617187f13be7a0f9942a096edce35